packages/mux-audio/test/player.test.js

```diff
@@ -35,6 +34,9 @@ describe('<mux-audio>', () => {
       });
     });
 
+    player.playbackId = 'vDpm5ygrRJgfIEPNIc02IJR4Trf3z00AiP';
```

Comment:
by setting the playback id after, it makes the test more reliable to be able to get a `loadstart` event.